pkg/fixer/fixes/fixes.go

```diff
@@ -22,6 +22,7 @@ func NewDefaultFixes() []Fix {
 		&NoWhitespaceComment{},
 		&DirectoryPackageMismatch{},
 		&NonRawRegexPattern{},
+		&PreferEqualsComparison{},
```

Comment:
Have you run 

```
go run main.go lint foo/wow.rego
go run main.go fix --force foo/wow.rego
```

?

There seems to be a slight difference in how the locations are matched which is causing the fix not to fire.